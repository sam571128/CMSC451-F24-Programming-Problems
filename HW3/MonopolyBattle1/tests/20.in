15 16 539
9 3
12 5
7 12
1 3
6 14
9 14
8 3
14 12
15 7
3 3
6 7
12 6
4 10
1 9
3 4
3 9
