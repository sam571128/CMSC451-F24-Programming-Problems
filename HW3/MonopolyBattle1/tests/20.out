-1 -1 10 15 16 16 14 -1 16 15 -1 16 -1 14 -1 